src/forklift/lift.py

```diff
@@ -161,72 +198,45 @@ def copy_data(pallets, all_pallets, config_copy_destinations):
                 except Exception:
                     log.error('%s might be in a corrupted state', destination_workspace, exc_info=True)
 
-                if source in destination_to_pallet:
-                    for pallet in destination_to_pallet[source]:
+                if source.lower() in destination_to_pallet:
+                    for pallet in destination_to_pallet[source.lower()]:
                         pallet.success = (False, str(e))
 
                 log.error('there was an error copying %s to %s', source, destination_workspace, exc_info=True)
 
-        for service in services:
-            log.debug('starting %s.%s', service[0], service[1])
-            status = lightswitch.turn_on(service[0], service[1])
+    for service in services_affected:
```

Comment:
One more attempt at coherence...

I think that there should be an `info` level log added here just like on line 155.